### DISCUSSION TITLE
채팅방 목록 불러올시 PlusChat

### DISCUSSION THREAD

**Discussion by MoonJR** [storycraft/node-kakao#444] [Q&A]
채팅방 목록 불러올시 PlusChat type이 잘 안불러와지는거 같은데 혹시 다른 분들도 그러신가요?

오픈채팅방이나 다른 부분은 정상적으로 불러와지는데 플러스친구는 잘 안가져와지네요 ;ㅁ;

talkClient.channelList.all() => 함수로 가져오고 있습니다.

**Comment by suuu111** [comment]
저도 방금 사용해봤지만 플러스챗이 불러와지지 않네요

**Comment by dlehdanakf** [comment]
해결되셨나요??
